internal/cli/deployment_list.go

```diff
@@ -127,6 +128,13 @@ func (c *DeploymentListCommand) Run(args []string) int {
 			c.project.UI.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
 			return ErrSentinel
 		}
+		if len(resp.Deployments) == 0 {
+			c.project.UI.Output(
+				fmt.Sprintf("No deployments found for application %q", app.Ref().Application),
```

Comment:
small nit, I believe `UI.Output` let's you format strings, so I think you can remove the `fmt.Sprintf` and just do `"your formatted string goes %s", "here"`?